codeSnippets/snippets/gradle-app/initial.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
```

Comment:
Is the reason for splitting the Gradle file into multiple so as to be able to reference them in the docs by sections?  If so, I wonder how this impacts the usability of a person that wants to just use this sample by clicking on the button to download it. It might feel overwhelmingly complex.

Reply:
The reason is having steps that are easier to understand. I can refer to the whole Gradle build file after.

Reply:
Right, so the project is going to have multiple Gradle build files then?

Reply:
Yes